README.md

```diff
@@ -4,6 +4,7 @@ Xcode iPhoneOS DeviceSupport (6.0 - 16.0)
 
 **Most recent releases:**</br>
 
+iOS 16.2 - [Download](https://github.com/filsv/iOSDeviceSupport/raw/master/16.1.zip) </br>
```

Comment:
@bhadjajay23 
The item described with the download link is not correct.
Item 16.2 is informing version 16.1 for downloading

Reply:
:( im waiting for this :(

Reply:
Has anyone checked it? It doesn't work for me on xcode 13.4.1